privaterelay/management/commands/invite_monitor_waitlist.py

```diff
@@ -1,30 +1,95 @@
+
     def handle(self, *args, **options):
+        limit = options['limit'][0]
+
         monitor_waitlist = MonitorSubscriber.objects.using('monitor').filter(
             waitlists_joined__email_relay__notified=False
-        )
+        )[:limit]
+
         print("adding %s invitations ..." % monitor_waitlist.count())
+
         for invitee in monitor_waitlist:
             try:
                 Invitations.objects.get(
                     email=invitee.primary_email,
                     active=True,
                 )
-                print("%s already has an active invitation" % invitee.primary_email)
+                print(
+                    "%s already has an active invitation" %
+                    invitee.primary_email
+                )
+                invitee.waitlists_joined['email_relay']['notified'] = True
+                invitee.save(update_fields=['waitlists_joined'])
             except Invitations.DoesNotExist:
                 print("adding %s to invitations" % invitee.primary_email)
-                Invitations.objects.create(
+                invitation = Invitations.objects.create(
                     email=invitee.primary_email,
                     active=True,
                     date_added=datetime.now(),
                     date_redeemed=None
                 )
+
+                context = {
+                    'email': invitee.primary_email,
+                    'current_domain': settings.SITE_ORIGIN,
+                }
+
+                # Send email invite
+                sl_message = BasicMessage()
+                sl_message.subject = (
+                    "You're Invited to the Private Relay Beta!"
+                )
+
+                sl_message.html_body = render_to_string(
+                    'emails/beta_invite_html_email.html',
+                    context,
+                )
+                sl_message.plain_text_body = render_to_string(
+                    'emails/beta_invite_text_email.txt',
+                    context,
+                )
+
+                relay_display_name, relay_from_address = parseaddr(
+                    settings.RELAY_FROM_ADDRESS
+                )
+                sl_message.from_email_address = EmailAddress(
+                    relay_from_address
+                )
+                sl_message.to_email_address.append(
+                    EmailAddress(invitee.primary_email)
+                )
+
+                sl_client = get_socketlabs_client()
+                response = socketlabs_send(sl_client, sl_message)
+
+                if not response.result.name == 'Success':
+                    logger.error('socketlabs_error', extra=response.to_json())
+
+                invitation.date_sent = datetime.now()
+                invitation.save(update_fields=['date_sent'])
+
+                invitee.waitlists_joined['email_relay']['notified'] = True
+                invitee.save(update_fields=['waitlists_joined'])
```

Comment:
```suggestion
                else:
                    invitation.date_sent = datetime.now()
                    invitation.save(update_fields=['date_sent'])

                    invitee.waitlists_joined['email_relay']['notified'] = True
                    invitee.save(update_fields=['waitlists_joined'])
```
values should be iff email was successfully delivered